config/test.exs

```diff
@@ -1,4 +1,4 @@
 use Mix.Config
 
 config :lob_elixir,
-  api_key: "test_5fae9e2d112690f0f8d534c5f9b4ad10257"
+  api_key: System.get_env("LOB_API_KEY")
```

Comment:
Technically we don't need this, but this causes race conditions in tests when the `LOB_API_KEY` env variable gets deleted for the `MissingAPIKeyError` test